source/javascripts/refills/dropdown.js

```diff
@@ -20,14 +20,10 @@ $(function() {
     }
   });
 
-  function parentIsMenu(element) {
-    return $(element).parents(menuSelector).length > 0;
-  }
-
   $('body').click(function(event) {
-    var target = $(event.target);
+    var button = findButton(event.target);
 
-    if (!(target.is(toggleSelector) || parentIsMenu(event.target))) {
+    if (!(button.is(toggleSelector) || parentIsMenu(event.target))) {
```

Comment:
We're checking for the case where we're clicking inside of the menu, which is not caught by the `findButton` function.